docs/pages/database-access/guides/clickhouse-self-hosted.mdx

```diff
@@ -121,95 +111,149 @@ you're running multiple services add `--output=stdout` to print the config in
+   `/var/lib/teleport/db.ca` on the Teleport Database Service host.
+
+1. Run a variation of the command above that uses the `--ca-cert-file`
+   flag. This configures the Teleport Database Service to use the CA certificate
+   at `db.ca` to verify traffic from the database:
+
+   ```code
+   $ sudo teleport db configure create \
+      -o file \
+      --token=/tmp/token \
+      --proxy=<Var name="teleport.example.com:443" /> \
+      --name=example-clickhouse \
+      --protocol=<Var name="protocol" /> \
+      --uri=clickhouse.example.com:8443 \
+      --ca-cert-file="/var/lib/teleport/db.ca" \
+      --labels=env=dev
+   ```
 
 (!docs/pages/includes/start-teleport.mdx service="the Teleport Database Service"!)
 
 </TabItem>
 <TabItem label="Kubernetes Cluster">
-  Teleport provides Helm charts for installing the Teleport Database Service in Kubernetes Clusters.
 
-  (!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+Teleport provides Helm charts for installing the Teleport Database Service in
+Kubernetes Clusters.
 
-Install the Teleport Kube Agent into your Kubernetes Cluster
-with the Teleport Database Service configuration.
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
 
-<Tabs>
-<TabItem label="HTTP">
+{/* Inlining docs/pages/includes/database-access/self-hosted-db-helm-install.mdx
+so we can use the Var component for the protocol*/}
 
-```code
-$ JOIN_TOKEN=$(cat /tmp/token)
-$ helm install teleport-kube-agent teleport/teleport-kube-agent \
-  --create-namespace \
-  --namespace teleport-agent \
-  --set roles=db \
-  --set proxyAddr=<Var name="teleport.example.com" description="Your Teleport Proxy Service or Teleport Cloud tenant address"/>:443 \
-  --set authToken=${JOIN_TOKEN?} \
-  --set "databases[0].name=example-clickhouse" \
-  --set "databases[0].uri=clickhouse.example.com:8443" \
-  --set "databases[0].protocol=clickhouse-http" \
-  --set "labels.env=dev" \
-  --version (=teleport.version=)
+Install a Teleport agent into your Kubernetes Cluster with the Teleport Database
+Service configuration. 
+
+Create a file called `values.yaml` with the following content. Update <Var
+name="teleport.example.com:443" /> to use the host and port of the Teleport Proxy
+Service and <Var name="JOIN_TOKEN" /> to the join token you created earlier:
+
+```yaml
+roles: db
+proxyAddr: <Var name="teleport.example.com:443" />
+# Set to false if using Teleport Community Edition
+enterprise: true
+authToken: <Var name="JOIN_TOKEN" />
+databases:
+  - name: example-clickhouse
+    uri: clickhouse.example.com:8443
+    protocol: <Var name="protocol" />
+labels:
+  env: dev
 ```
 
-</TabItem>
-<TabItem label="Native (TCP)">
+To configure the Teleport Database Service to trust a custom CA:
+
+1. Export a CA certificate for the custom CA and make it available at `db.ca` on
+   your workstation.
+
+1. Create a secret containing the database CA certificate in the same namespace
+   as Teleport using the following command:
```

Comment:
CA certificates are public info - why create a secret? (i'm not kube savvy btw)

Reply:
This is based on the guidance for using `ca_cert_file` with `teleport-kube-agent` in the Helm chart reference: https://goteleport.com/docs/reference/helm-reference/teleport-kube-agent/#databases

I suppose we could also use a ConfigMap instead of a secret and create a volume from it in the same way—I think it would be functionally the same.

Reply:
oh ok. I think it's fine to leave it as a secret since we already do that elsewhere. I also noticed CrunchyData's PG Operator guide does the same thing for CA certs, I just didn't understand why

Reply:
There's nothing blocking you from putting CA certs in a configmap, this is what kube automatically does for its CA cert and makes sense from a security point of view.

Putting the CA certs in a secret is a common practice in kube. I think this is because you usually also have private material (key/cert for a TLS webserver, or the CA private key when it's a private CA managed by cert-manager in the same cluster).

Another reason is that users have very limited understanding of asymmetric cryptography and often do unsafe things like sending the CA private key in a context where only the cert is needed (I've seen this many times in the wild, including in support calls with customers). I'd be in favor of keeping everything in a secret so the failure mode is not that bad.

Reply:
Makes sense, thanks!